crates/wasm-smith/src/config.rs

```diff
@@ -254,6 +254,11 @@ pub trait Config: Arbitrary + Default + Clone {
     fn max_nesting_depth(&self) -> usize {
         10
     }
+
+    /// Returns the maximal effective size of any type generated by wasm-smith.
```

Comment:
What are the units? I think this should be documented, even if it is just "*approximately* the number of items imported or exported by a module or instance and the number of parameters/results used by functions" to give users a sense of what magnitude number to use.